test/utils/probability/test_lin_ess.py

```diff
@@ -462,3 +462,44 @@ def test_multivariate(self):
             self.assertEqual(X_high_d.shape, torch.Size([16, d]))
             self.assertTrue(sampler._is_feasible(X_high_d.T).all())
             self.assertEqual(sampler.lifetime_samples, num_samples)
+
+    def test_batch_mcmc(self):
+        d = 5
+        for i in range(2):
+            torch.manual_seed(i)
```

Comment:
This doesn't need to be checked for both seeds every time. One option would be to confirm it works for a range of seeds (say 0-10) and then randomly choose one of those seeds each time the test is run.

Reply:
Will remove the for loop.